src/commands/block/deploy.ts

```diff
@@ -69,83 +73,102 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
     const codeDirectory =
       flags.context ?? (await ux.prompt("Where is your code checked out?"));
     const infraDirectory = "generated";
-    const region = diggerConfig.aws_region;
+    const region = flags.region
     const terraformOutputs = await tfOutput(infraDirectory);
-    const url = terraformOutputs[args.name].value.lb_dns;
-    const ecrRepoUrl = terraformOutputs[args.name].value.docker_registry_url;
-    const ecsClusterName = terraformOutputs[args.name].value.ecs_cluster_name;
-    const ecsServiceName = terraformOutputs[args.name].value.ecs_service_name;
+
+    const blockRegions = region === undefined || region === null ? Object.keys(diggerConfig.blocks.find((block: any) => block.name === args.name).aws_regions) : [region];
+
+    const configPerRegion: {[region: string]: {[key: string]: any}} = {}
     const awsProfile = "default";
+    for (const region of blockRegions) {
+      const moduleName = `${args.name}_${region}`;
+      const lbUrl = terraformOutputs[moduleName].value.lb_dns;
+      const ecrRepoUrl = terraformOutputs[moduleName].value.docker_registry_url;
+      const ecsClusterName = terraformOutputs[moduleName].value.ecs_cluster_name;
+      const ecsServiceName = terraformOutputs[moduleName].value.ecs_service_name;
```

Comment:
For you to decide if this would look nicer:

```
const {
  lb_dns: lbUrl,
  docker_registry_url: ecrRepoUrl,
  ecs_cluster_name: ecsClusterName,
  ecs_service_name: ecsServiceName
} = terraformOutputs[moduleName].value
```